portal-kernel/src/com/liferay/document/library/kernel/service/packageinfo

```diff
@@ -1 +1 @@
-version 6.2.0
\ No newline at end of file
+version 6.3.0
```

Comment:
We need to avoid mixing different things in the same commits to make the review easier. I would put this in a baseline commit